scripts/patterns-reorg/entries.py

```diff
@@ -84,6 +84,7 @@
         ),
     ),
     Entry("Hello world", slug="/hello-world", from_file="start/hello-world.ipynb"),
+    Entry("Latest updates", slug="/latest-updates", from_file="start/lates-updates.mdx"),
```

Comment:
@Eric-Arellano just double-checking that I added this to the entries.py file correctly?

Reply:
Other than typo. Thanks for adding this!

```suggestion
    Entry("Latest updates", slug="/latest-updates", from_file="start/latest-updates.mdx"),
```

Reply:
Eek! Thanks!